source/concepts/30-message-sequence.md

```diff
@@ -0,0 +1,32 @@
+## Message Sequence
+
+This section uses the overall message flow to show how and when the NodeX Agent encrypts and decrypts messages as they are communicated between devices.
```

Comment:
The diagram below describes the sequence of Verifiable Messages. Therefore, it might be better to express this as "This section describes the overall message flow of how and when the NodeX Agent sends and receives verifiable messages between devices."

In NodeX v2, there are two types of message flows: Verifiable Message and E2E Encrypted (VM). It might be more considerate to separate the sequences to show these two types.